server/webapp/WEB-INF/rails/webpack/views/components/header_panel/css/index.scss

```diff
@@ -0,0 +1,48 @@
+/*!
```

Comment:
Please no `/*!` that causes the comment to be included in the output CSS. Google for "loud css/scss comment". Preferably add a lint for it :)